cli/create.go

```diff
@@ -49,6 +49,10 @@ func (cc *CreateCommand) runCreate(args []string) error {
 	config := cc.config()
 
 	config.Image = args[0]
+	if config.Image != "" && !strings.Contains(config.Image, ":") {
+		config.Image += ":latest"
```

Comment:
Actually I am not totally on this change. Here I have an example `localhost:80/allen/ubuntu`. I think we need to latest tag as well.

Reply:
You are right. Do you think below code can work?
```golang
if config.Image != "" {
	image := strings.Split(config.Image, ":")
	if len(image) == 1 || strings.Contains(image[1], "/") {
		config.Image += ":latest"
	}
}
```

Reply:
The fix is important. And I think we can use the pkg reference https://github.com/alibaba/pouch/blob/master/pkg/reference/reference.go to parse image?
WDYT?

And this will make code more well-encapsulated? @xwjdsh

Reply:
Cool, changed as your request.